dev/faq/tidb-in-kubernetes.md

```diff
@@ -0,0 +1,22 @@
+---
+title: TiDB FAQs in Kubernetes
+summary: Learn about TiDB FAQs in Kubernetes.
+category: FAQ
```

Comment:
The change in https://github.com/pingcap/docs-cn/pull/1601 will be addressed in another PR?

Reply:
@danielzhangQD These updates will be added to the troubleshooting section.

Reply:
@danielzhangQD These updates will be added to the troubleshooting section.